dt/nova/nova-three-cells/edpm/nodeset/nova_cell_service.yaml

```diff
@@ -0,0 +1,21 @@
+---
+apiVersion: dataplane.openstack.org/v1beta1
+kind: OpenStackDataPlaneService
+metadata:
+  name: nova
```

Comment:
Don't we have this automatically created by the openstack-operator based on the built in default service definitions in https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml ?

Does other DT/VAs define the nova service the same way so you are just following a pattern or is it something specific to this DT?

Is it only needed here to be able to create a copy of it and patch is to define the cell2 nova service? If so then I'm wondering why we don't simply store the cell2 nova service instead of patching of a copy. I could imagine that we want to keep cell1 and cell2 nova service consistent but I think we can only do so if we somehow able to reuse the default definition from https://github.com/openstack-k8s-operators/openstack-operator/blob/main/config/services/dataplane_v1beta1_openstackdataplaneservice_nova.yaml as that is the source of truth afaik.

Reply:
Its the latter, you are correct the service is defined without needing to be explicitly defined in the DT.  But the only mechanism that would allow me to create a second service for cell2 would be to define it within the DT.  Let me try moving this to purely cell2 though instead of as a copy and patch.